src/index/api.rs

```diff
@@ -119,7 +118,11 @@ impl Api {
             downloads: i32,
         }
 
-        let response: Response = self.client.get(url).send()?.error_for_status()?.json()?;
+        let response: Response = retry(
+            || Ok(self.client.get(url.clone()).send()?.error_for_status()?),
+            3,
+        )?
+        .json()?;
```

Comment:
Actually, should we also retry the owners request? (probably extract this line out to a separate method).

Reply:
addewd in #2073 , very valid point. I missed that